src/schema/v2/External/ExternalAuctionHouse.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  GraphQLInt,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from "graphql"
+import { ResolverContext } from "types/graphql"
+
+export interface Response {
```

Comment:
Types here we just generated with QuickType then validated against the models in Galaxy.